lib/policies/key-auth/index.js

```diff
@@ -1,5 +1,6 @@
-'use strict';
-
 module.exports = {
-  policy: require('./keyauth')
+  policy: require('./keyauth'),
+  schema: {
+    // TODO: add schema
```

Comment:
##### Options Reference

* `apiKeyheader`:
  - 'Authorization'
  - the name of the header that should contain the API key
* `apiKeyHeaderScheme`:
  - 'apiKey'
  - the enforced scheme in the header
* `apiKeyHeader`:
  - 'Authorization'
  - name of the header that should contain api key
* `apiKeyHeaderScheme`:
  - 'apiKey'
  - enforce a schema in the header
* `disableHeaders`:
  - false
  - disable apikey lookup in headers
* `disableHeadersScheme`:
  - false
  - disable verification of scheme in header
* `apiKeyField`:
  - 'apiKey'
  - name of field to check in query parameter
* `disableQueryParam`:
  - false
  - set to true to disable api key lookup in query string